src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/IUnitTestingIncrementalAnalyzer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal interface IUnitTestingIncrementalAnalyzer
```

Comment:
Use the `Implementation` suffix to indicate that this interface will be implemented in external code

```suggestion
    internal interface IUnitTestingIncrementalAnalyzerImplementation
```